src/client.rs

```diff
@@ -111,6 +111,71 @@ impl Client {
             .await?)
     }
 
+    /// Patches subresource status in a given Resource using apply strategy.
+    /// The subresource status must be defined beforehand in the Crd.
+    pub async fn apply_patch_status<T, S>(&self, resource: &T, status: &S) -> OperatorResult<T>
+    where
+        T: Clone + DeserializeOwned + Meta + Resource,
+        S: Serialize,
+    {
+        let new_status = Patch::Apply(serde_json::json!({
+            "apiVersion": T::API_VERSION,
+            "kind": T::KIND,
+            "status": status
+        }));
+
+        Ok(self
+            .patch_status(resource, new_status, &self.patch_params)
+            .await?)
+    }
+
+    /// Patches subresource status in a given Resource using merge strategy.
+    /// The subresource status must be defined beforehand in the Crd.
+    pub async fn merge_patch_status<T, S>(&self, resource: &T, status: &S) -> OperatorResult<T>
+    where
+        T: Clone + DeserializeOwned + Meta + Resource,
+        S: Serialize,
+    {
+        let new_status = Patch::Merge(serde_json::json!({
+            "apiVersion": T::API_VERSION,
+            "kind": T::KIND,
```

Comment:
I'm 99% certain that these two are only required for the `apply` patch but they don't seem to hurt either....

Reply:
Yes just tested you were right. Removed it! One last commit :)